client/src/components/FlipCard.js

```diff
@@ -0,0 +1,125 @@
+import React from "react";
+import styled from "styled-components/macro";
+import PropTypes from "prop-types";
+
+const FlipCardContainer = styled.div`
+  background-color: transparent;
+  margin: 8vw;
```

Comment:
8vw seems a little odd as a margin value, maybe use rem or %

Reply:
Rem sound good, just tested it and it works perfectly fine, thank you :)